nixos/modules/misc/ids.nix

```diff
@@ -638,6 +639,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      bitwarden_rs = 315;
```

Comment:
would a systemd dynamic user do the trick ?
That is, do other services need access to files owned by this user ?

Reply:
I did not know about systemd dynamic users. I just read about them and I guess that I could change the configuration to use it, since indeed only `bitwarden_rs` would need access, but I think it best to keep the `user` and `group` options available in case of `dataFolder` is changed from its default location.

Reply:
In any case, static uid/gid's aren't needed, not assigning them in the module makes NixOS choose one on its own, which will be constant across the lifetime of a NixOS installation.

Reply:
Ah, I did not know this, so it would be safe to remove `uid`/`gid` in `extraUsers`/`extraGroups` and remove the entries in `ids.nix`?

Reply:
Yup

Reply:
I have updated the PR to no longer use static uid/gid's.